deployment/k8s-deploy.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+export IMAGE_TAG=v3.0
```

Comment:
Those should also not be in the script.
These can be passed also as parameter(s).

Reply:
Done